clients/pkg/promtail/server/server.go

```diff
@@ -36,7 +36,7 @@ type Server interface {
 }
 
 // Server embed weaveworks server with static file and templating capability
-type server struct {
+type PromtailServer struct {
```

Comment:
Why is PromtailServer exported? It doesn't seem to be used anywhere else directly.